examples/Frank2Example4/src/main/configurations/Hello World/ConfigurationHelloLines.xml

```diff
@@ -54,9 +54,6 @@
 				onlyIfSessionKey="receivedRootElement"
 				onlyIfValue="Envelope"
 			/>
-			<Exits>
-				<Exit name="READY" state="SUCCESS" flow:x="1300" flow:y="1000"/>
-			</Exits>
 			<XsltPipe
```

Comment:
Deze staat er denk ik bij zodat de exit op de juiste plek staat in de Frank!Flow grafische weergave, kun je dat checken?

Reply:
@jacodg, ik begrijp je uitleg dat de Frank!Flow alleen exits kan tekenen als ze expliciet zijn. Daarom snap ik dat de exits hier niet weg kunnen.